packages/fxa-settings/src/components/ResetPasswordWarning/index.tsx

```diff
@@ -49,50 +58,80 @@ const ResetPasswordWarning = () => {
           aria-label={expanded ? 'Collapse warning' : 'Expand warning'}
         />
       </summary>
-      <div className="flex flex-col ps-8 pt-4 pb-2 gap-4">
-        <div className="flex items-start gap-2">
-          <IconSyncDevice
-            role="img"
-            className="flex-initial"
-            aria-hidden={true}
-          />
-          <div className="flex flex-col flex-1 -mt-1 gap-1">
-            <FtlMsg id="password-reset-previously-signed-in-device">
-              <p className="font-semibold">
-                Have a device where you previously signed in?
+      <div className="flex flex-col pt-4 pb-2 gap-4 text-xs leading-snug">
+        {locationState.recoveryKeyExists !== false && (
+          <div className="flex">
+            <IconSyncDevice
+              role="img"
+              className="me-4 mt-[2px]"
+              aria-hidden={true}
+            />
+            <div className="flex flex-col flex-1 gap-1">
+              <FtlMsg id="password-reset-warning-have-key">
+                <p className="font-semibold text-sm">
+                  Have an account recovery key?
+                </p>
+              </FtlMsg>
+              <FtlMsg id="password-reset-warning-use-key-link">
+                <Link
+                  to={`/account_recovery_confirm_key${searchParams}`}
```

Comment:
Do we need `searchParams || ''` here if this is undefined, or `?{searchParams}`? (I didn't test, just have seen it be wonky myself, maybe `Link` handles this better than `navigate`)